content/en/docs/ambient/install/istioctl/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Install with istioctl
+description: Install Istio with support for ambient mode using the istioctl command line tool.
+weight: 10
+keywords: [istioctl,ambient]
+owner: istio/wg-environments-maintainers
+test: yes
+---
+
+{{< tip >}}
+Follow this guide to install and configure an Istio mesh with support for ambient mode.
+If you are new to Istio, and just want to try it out, follow the
+[quick start instructions](/docs/ambient/getting-started) instead.
+{{< /tip >}}
+
+This installation guide uses the [istioctl](/docs/reference/commands/istioctl/) command-line
+tool to provide rich customization of Istio installation options. It has user input validation
+to help prevent installation errors and customization options to override any aspect of the
+configuration.
+
+Using these instructions, you can select any one of Istio's built-in
+[configuration profiles](/docs/setup/additional-setup/config-profiles/)
+and then further customize the configuration for your specific needs.
```

Comment:
Ditto - either advertise the same profile stuff on the Helm page (this is not unique to `istioctl`, the same profiles are used for both options) or drop it in both places.

Reply:
This will need to be cleaned up in the context of IstioOperator no longer being a custom resource.